platform/mds/mds/src/main/java/org/motechproject/mds/listener/records/BaseListener.java

```diff
@@ -24,6 +32,9 @@
  */
 public abstract class BaseListener<T> implements ServiceListener {
 
+    private static final String MDS_ENTITIES_SYMBOLIC_NAME = "org.motechproject.motech-platform-dataservices-entities";
```

Comment:
This constant already exists in the Constants class. I've mentioned this in the previous comment.